<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pgintegrity</class>
 <widget class="QWidget" name="pgintegrity">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>467</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTreeView" name="alltable_qtview">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>221</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
border-width:1px;
border-color:rgb(190,190,190);
background-color: rgb(255, 255, 255);
border-radius: 6px; </string>
   </property>
  </widget>
  <widget class="QLabel" name="verfy_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(78, 160, 201);
font: 11pt &quot;Agency FB&quot;;
border-style: hidden;</string>
   </property>
   <property name="text">
    <string>选择待验证数据表：</string>
   </property>
  </widget>
  <widget class="QTreeView" name="verifyedtable_qtview">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>221</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
border-width:1px;
border-color:rgb(190,190,190);
background-color: rgb(255, 255, 255);
border-radius: 6px; </string>
   </property>
  </widget>
  <widget class="QPushButton" name="addtable_btn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);
background-color: rgb(78, 160, 201);
border-radius: 5px; 
</string>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removetable_btn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);
background-color: rgb(78, 160, 201);
border-radius: 5px; 
</string>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="output_textedit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>511</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
border-width:1px;
border-color:rgb(190,190,190);
background-color: rgb(255, 255, 255);
border-radius: 6px; </string>
   </property>
  </widget>
  <widget class="QLabel" name="output_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(78, 160, 201);
font: 11pt &quot;Agency FB&quot;;
border-style: hidden;</string>
   </property>
   <property name="text">
    <string>控制台输出：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="verify_btn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>430</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);
background-color: rgb(78, 160, 201);
border-radius: 5px; 
</string>
   </property>
   <property name="text">
    <string>验证</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>430</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);
background-color: rgb(78, 160, 201);
border-radius: 5px; 
</string>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
